Jenkinsfile

```diff
@@ -60,19 +60,15 @@ pipeline {
                             filename 'Dockerfile.sles.12.3'
                             label 'docker_runner'
                             args '--privileged=true'
-                            additionalBuildArgs '--build-arg UID=$(id -u)'
+                            additionalBuildArgs '--build-arg UID=$(id -u)' +
+                                                ' --build-arg CACHEBUST=' +
+                                                currentBuild.startTimeInMillis
```

Comment:
Oh and you will only get the output if you cachebust the layer?

I have been considering that we might want to have multiple CACHEBUST values for differing cadences.  The `currentBuild.startTimeInMillis` is an obvious one to force something to happen on every build, but we might want to have daily and weekly ones also, and perhaps even monthly maybe.  Who knows.

Nothing to be implemented now I don't think.  It can mature once we have the submodule/reference to work with.